doc/00_Linee guida interoperabilità tecnica/02_ambito-di-applicazione.rst

```diff
@@ -4,22 +4,20 @@ Ambito di applicazione
 Le Linee Guida sull’interoperabilità tecnica delle Pubbliche 
 Amministrazioni sono redatte in ottemperanza:
 
--  alla lettera a comma 2 dell’articolo 14-bis del DECRETO LEGISLATIVO 
-   7 marzo 2005, n. 82 (Codice dell'amministrazione digitale), che 
-   individua le funzioni di AgID tra cui: "emanazione di Linee [...] 
+-  alla lettera a comma 2 dell’articolo 14-bis del Decreto Legislativo 
+   7 marzo 2005, n. 82 (Codice dell'Amministrazione Digitale, CAD), che 
```

Comment:
@mfortini da qui https://docs.italia.it/italia/designers-italia/writing-toolkit/it/bozza/index.html credo che vada maiuscola solo la prima lettera. Giusto invece correggere `Decreto legislativo`

Reply:
@ioggstream @mfortini ho provato ad approfondire il tema dei riferimenti esterni, in base al seguente https://www.senato.it/application/xmanager/projects/senato/file/repository/istituzione/regole_testi_legislativi.pdf evidenzio che in diversi punti è indicato l'uso di "decreto legislativo"

Reply:
Giusto l'approfondimento dal writing toolkit. Io mi ero confrontato col sito di AgID che mette le maiuscole, potremmo cercare una linea comune https://www.agid.gov.it/it/agenzia/strategia-quadro-normativo/codice-amministrazione-digitale

Reply:
anche nel riferimento da te indicato si usa "decreto legislativo"